### ISSUE TITLE
callback not working

### ISSUE THREAD

**Issue by Pako2** [guglielmino/pushetta-py#3]
Hi!
Thank you for your interesting project. However, I have one problem with it.
I can not put into operation callback by your example (def my_callback(payload): ...)
from https://github.com/guglielmino/pushetta-py.
Everything seems to be okay, but nothing happens when I do I send a message.
You're sure it should work this way?

Thanks, Lubos

**Comment by guglielmino** [comment]
Hi Lubos,

I made a quick try right now and all works fine.
Can You share the code?

Thanks
Fabrizio

**Comment by Pako2** [comment]
[pshtt.zip](https://github.com/guglielmino/pushetta-py/files/116071/pshtt.zip)

Hi Fabrizio !
I thank you for your prompt response.
My script is attached.
Of course, this is just a test version. It runs in the Raspberry Pi.
I get only  "Pushetta listener START" and then nothing.

BTW - even Chrome extension does not work for me.
When I open the page in the background, there is a visible error message:
Failed to load resource: net::ERR_INSECURE_RESPONSE

Edit:
I found that the certificate www.pushetta.com has expired 16. 1. 2016.
Maybe that's the problem?

Keep up the good work !

Best regards,
Lubos

**Comment by guglielmino** [comment]
Hi Lubos,

Looking Your script there is an error, this line
p=Pushetta("API_KEY")
Have to be
p=Pushetta(API_KEY)
Whitout double quotes. 

About the problem with chrome it happens because is expired a https certificate and I have to update it.

Bye
Fabrizio

**Comment by Pako2** [comment]
Hi Fabrizio,
I'm very sorry, it is completely obvious.
It is a disgrace that I did not find it myself.
However, I copied it from your example, and I did not expect that there is a bug, and for that reason I tried to find fault elsewhere.

Thanks, Lubos

**Comment by Pako2** [comment]
Hi Fabrizio,
I discovered that the script works only if the channel is marked as "public".
Why is this so?

Thanks,
Lubos

**Comment by guglielmino** [comment]
When You use a private channel an email to approve subscription is sent to the channel owner for every new device which try to subscribe the channel itself. Have You received the email? In this email there is a link to follow to approve the request.

**Comment by Pako2** [comment]
No, I've got only one email (it was when I created my pushetta account).
Then I also, I set up a subscription for two mobile phones to my test channel, but no e-mail arrived (this is probably another case, the subscription is alright). Or it is possible that at the time of the creation of the subscription, the channel has been marked as public.
When I try subscription to my **privat** channel with my python script, no e-mail arrives.

I have a feeling that something is missing in the API documentation:
- How to create a "device" (other than a smartphone - for example Raspberry Pi)
- How to delete this device

**BTW:**
I found another bug in your example script. There are missing **import datetime**.

**Comment by Dishant-Tawade** [comment]
for some reason my raspberry pi does not have a module named 'pushetta'. Can someone tell me what I have to do to fix this problem.
